benchmark/benchmark_performance/smart_pool/src/smart_pool_benchmark.cpp

```diff
@@ -0,0 +1,156 @@
+#include <iostream>
+#include <thread>
+
+#include <benchmark/benchmark.h>
+
+#include "point3d_cloud.h"
+#include <klepsydra/core/smart_object_pool.h>
+
+static void BM_SmartPoolAcquireInt(benchmark::State &state)
```

Comment:
Remove all tests that are copy from kpsr-core. I.e., this one.

Reply:
Done.